### DISCUSSION TITLE
Any plan for implementation as plugin?

### DISCUSSION THREAD

**Discussion by jihoon-yun** [HiIAmMoot/RuntimeBPs#34] [General]
I just want to know if there is any plan or progress on implementing the project as the plugin, since I am working on it personally.

**Comment by HiIAmMoot** [comment]
There are plans to do so. I'm trying to figure out how to fix the circular references issue first. Because the struct FNodeVarArgs is supposed to have flexible variable types, including those declared and defined inside your project source. Right now I managed to make it work as a separate module, I want to clean up some things first before I attempt making it a plugin. I'll make an issue.
